frameworks/cassandra/universe/marathon.json.mustache

```diff
@@ -45,13 +45,15 @@
     "CASSANDRA_DISK_TYPE": "{{nodes.disk_type}}",
     "TASKCFG_ALL_CASSANDRA_HEAP_SIZE_MB": "{{nodes.heap.size}}",
     "TASKCFG_ALL_CASSANDRA_HEAP_NEW_MB": "{{nodes.heap.new}}",
+    "CASSANDRA_JAVA_URI": "{{resource.assets.uris.cassandra-jre-tar-gz}}",
     "CASSANDRA_HEAP_GC": "{{nodes.heap.gc}}",
     "PLACEMENT_CONSTRAINT": "{{nodes.placement_constraint}}",
     {{#service.virtual_networks}}
     "ENABLE_VIRTUAL_NETWORK": "yes",
     "VIRTUAL_NETWORK_NAME": "{{nodes.virtual_network}}",
     "CNI_PLUGIN_LABELS": "{{nodes.cni_plugin_labels}}",
     {{/service.virtual_networks}}
+    "TASKCFG_ALL_JAVA_HOME": "jre1.8.0_131/",
```

Comment:
To avoid dealing with updating this when the JRE version changes, should instead set `JAVA_HOME` like this within the `cmd`:

```
export JAVA_HOME=$(ls -d $MESOS_SANDBOX/jre*/)
... rest of cmd here ...
```